README.md

```diff
@@ -13,5 +13,5 @@ Website of 24chevres.com
   - folks in 24chevres organization will vote the PR (with :+1: and :-1: on the PR's description)
   - prefix the PR's description with `[RDY]` when the PR is ready to be merged
   - wait 24 hours after the last commit
-  - if vote result is positive, the PR will be merged
+  - if vote result is positive (2 more :+1: than :-1:), the PR will be merged
```

Comment:
unclear

Reply:
How about that?
> the PR will be merged when the majority +1 voted positively

Reply:
it's bad :D halp!

Reply:
“If there are more :+1: reactions than :-1:, the PR will be merged”?

Reply:
2 more are needed

2 :+1: and 1 :-1:  won't merge

Reply:
Where is @msavournin when we need him?

Reply:
the pull request is validated if there is 2 (+1) more than there are (-1)